.github/workflows/push.yml

```diff
@@ -0,0 +1,59 @@
+name: Push workflow
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push:
+    name: openrate-consumer
```

Comment:
Give proper job name?

Reply:
Oops, I copied this in by accident as I copied this workflow from another project i worked on